sera.go

```diff
@@ -91,24 +98,33 @@ func realMain() (int, error) {
 		return ExitBadArguments, err
 	}
 
+	if timeout > 0 && conf.IgnoreUnacquirable {
+		fmt.Printf("Warning: non-zero timeout (%ds) in ignore-unacquirable could be awkward.\n", timeout/time.Second)
+	}
+
 	// create db object
 	db, err := sql.Open("mysql", conf.Server)
 	if err != nil {
 		return ExitLockFailed, err
 	}
 	defer db.Close()
 
-	mutex := NewMysqlMutex(db, keyName, timeout)
+	mutex := NewMysqlMutex(&MysqlConnection{DB: db}, keyName, timeout)
 
 	// Try to get the lock, block until we get the lock or we reached the timeout value
 	if err = mutex.Lock(); err != nil {
+		// In first past the post mode, we are happy for subsequent agents to not execute.
+		if _, ok := err.(ErrLockTaken); ok && conf.IgnoreUnacquirable {
+			fmt.Println("Unable to acquire lock, but reporting success.")
```

Comment:
I would drop this, It would cause a lot of noise in logs etc for an expected behaviour.